network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/advanced_table_block.html

```diff
@@ -0,0 +1,51 @@
+{% load wagtailcore_tags %}
+
+<div class="article-table-block table-responsive">
+  <table class="table">
+    {% if self.caption %}
+      <caption>{{ self.caption }}</caption>
+    {% endif %}
+
+    {# Loop through every row #}
+    {% for row in self.table %}
+      {# If the "header" is checked, the first row will be a <thead> with a blue background #}
```

Comment:
In case we decide to change the highlight colour in the future, could we update the comment from "blue" background to "highlighted" background?

Reply:
Good suggestion. Absolutely, yes, let's do this.